src/storage/conversion/ChainedConverter.ts

```diff
@@ -34,31 +35,19 @@ export class ChainedConverter extends TypedRepresentationConverter {
     return this.converters[this.converters.length - 1];
   }
 
-  public async getInputTypes(): Promise<Record<string, number>> {
+  public async getInputTypes(): Promise<ValuePreferences> {
     return this.first.getInputTypes();
   }
 
-  public async getOutputTypes(): Promise<Record<string, number>> {
+  public async getOutputTypes(): Promise<ValuePreferences> {
     return this.last.getOutputTypes();
   }
 
-  public async canHandle(input: RepresentationConverterArgs): Promise<void> {
-    // We assume a chain can be constructed, otherwise there would be a configuration issue
-    // So we only check if the input can be parsed and the preferred type can be written
-    const inTypes = this.filterTypes(await this.first.getInputTypes());
-    const outTypes = this.filterTypes(await this.last.getOutputTypes());
-    supportsConversion(input, inTypes, outTypes);
-  }
```

Comment:
What is the reason for removing the canHandle function? This would prevent us from having multiple ChainedConverters that support different chains (because we have several converters that use different intermediate types for example).

Reply:
> What is the reason for removing the canHandle function?

Base class does it now.

Reply:
Ah I missed that